fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2220,7 +2306,41 @@ public boolean isIndexDisabled(@Nonnull String indexName) {
      * Reason that an index is being rebuilt now.
      */
     public enum RebuildIndexReason {
-        NEW_STORE, FEW_RECORDS, COUNTS_UNKNOWN, REBUILD_ALL, EXPLICIT, TEST
```

Comment:
If this becomes a protobuf `enum` there will be no need for the code.

Reply:
Do you want this to go to protobuf? It doesn't seem necessary for this change, but seems easy to me.